<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShermanMainWin</class>
 <widget class="QMainWindow" name="ShermanMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShermancitoMainWin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/art.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionTurn_Left"/>
   <addaction name="actionTurn_Right"/>
   <addaction name="actionDelete"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionTurn_Left">
   <property name="text">
    <string>Turn Left</string>
   </property>
   <property name="toolTip">
    <string>Turn Left (Press &quot;,&quot;)</string>
   </property>
  </action>
  <action name="actionTurn_Right">
   <property name="text">
    <string>Turn Right</string>
   </property>
   <property name="toolTip">
    <string>Turn Right (Press &quot;.&quot;)</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../resources/art.qrc">
     <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/art.qrc"/>
 </resources>
 <connections/>
</ui>
